packages/expo-prevent-screen-capture/README.md

```diff
@@ -0,0 +1,33 @@
+# expo-prevent-screen-capture
+
+This module allows you to prevent users from screen recording or taking screenshots of your app.
+
+> Currently, taking screenshots on iOS cannot be prevented. This is due to underlying OS limitations.
+
+# API documentation
+
+# Installation in managed Expo projects
+
+For managed [managed](https://docs.expo.io/versions/latest/introduction/managed-vs-bare/) Expo projects, please follow the installation instructions in the [API documentation for the latest stable release](https://docs.expo.io/versions/latest/sdk/prevent-screen-capture/).
+
+# Installation in bare React Native projects
+
+For bare React Native projects, you must ensure that you have [installed and configured the `react-native-unimodules` package](https://github.com/unimodules/react-native-unimodules) before continuing.
+
+### Add the package to your npm dependencies
+
+```
+expo install expo-calendar
+```
+
+### Configure for iOS
+
+```
+pod install
```

Comment:
I've heard there is something called `npx pod-install` that is you know, great and all. Do we want to use it in our docs? 🤔